dashboard/db/migrate/20210908195236_update_backpack_storage_apps_value.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateBackpackStorageAppsValue < ActiveRecord::Migration[5.2]
+  def change
+    Backpack.all.each do |backpack|
```

Comment:
I'm wondering if we should just look in the storage_apps table for all rows with `project_type='backpack'` (and avoid looking at the backpacks table at all).  It looks like there's an index on project_type so this should be safe to run even in production.

Reply:
Yes! Looks like that results in the same results, I'll do that

Reply:
Updated!